### ISSUE TITLE
False positive: DL3022 COPY --from should reference a previously defined FROM alias

### ISSUE THREAD

**Issue by fniessink** [hadolint/hadolint#271]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Hadolint does not complain about artifacts copied from externals images

### Actual behavior

Hadolint complains about artefacts copied from external images with error message `DL3022 COPY --from should reference a previously defined FROM alias`

### Steps to reproduce the behavior

Using this Dockerfile:

```Dockerfile
FROM alpine:3.8
  
COPY --from=hadolint/hadolint /bin/hadolint /usr/local/bin
```

And hadolint version 1.13:

```bash
$ docker run --rm hadolint/hadolint hadolint --version
Haskell Dockerfile Linter v1.13.0-2-g8623159
```

Hadolint complains:

```bash
$ docker run --rm -i hadolint/hadolint < Dockerfile
/dev/stdin:3 DL3022 COPY --from should reference a previously defined FROM alias
```

Tested on macOS:
```bash
$ docker --version
Docker version 18.06.1-ce, build e68fc7a
```

**Comment by lorenzo** [comment]
This is a duplicate of https://github.com/hadolint/hadolint/issues/197

Copying from an external image is something that in my view should still be done in 2 steps, first a `FROM` and then `COPY`. This way we can also assert that you are copying from a specific tag and it is in my experience easier to read.

**Comment by fniessink** [comment]
Thanks, I missed that issue when searching.
